benchmark/util/type-check.js

```diff
@@ -38,7 +38,7 @@ function main(conf) {
   const util = process.binding('util');
   const types = require('internal/util/types');
 
-  const n = (+conf.millions * 1e6) | 0;
+  const n = (+conf.n * 1e6) | 0;
```

Comment:
This would increase `n` from `5 * 1e6` to `1e7 * 1e6`, which is going to cost more time if the benchmark is run normally. Can you change the default n to `5e6` and use that value as-is(without multiplying) instead?

Reply:
Ah yes, I agree that is a better solution.